tool/src/ExplainDiffHandler.cs

```diff
@@ -76,7 +77,19 @@ public async Task<int> InvokeAsync(InvocationContext context)
             return 0;
         }
     }
-    private static async Task WriteToGitHubOutput(Difference[] results)
+    private static async Task WriteExplanationToFileAsync(string explanationResult)
+    {
+        var directory = Directory.GetCurrentDirectory();
+        var explanationFilePath = System.IO.Path.Combine(directory, "explanations.txt");
+        await File.WriteAllTextAsync(explanationFilePath, explanationResult).ConfigureAwait(false);
+        var githubOutputFile = Environment.GetEnvironmentVariable("GITHUB_OUTPUT", EnvironmentVariableTarget.Process);
+        if (!string.IsNullOrWhiteSpace(githubOutputFile))
+        {
+            using var textWriter = new StreamWriter(githubOutputFile!, true, Encoding.UTF8);
```

Comment:
we should refactor the creation of the writer/writing to the output to avoid duplication